reference/customers.v3.yml

```diff
@@ -712,6 +713,86 @@ paths:
           headers: {}
       tags:
         - Customer Addresses
+  /customers/validate-credentials:
+    post:
+      tags:
+        - Customer Validate Credentials
+      description: Validate a customer credentials - This endpoint has special rate limiting protections to protect against abuse
+      summary: Validate a customer credentials
+      operationId: CustomerValidateCredentials
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ValidateCustomerCredentialsObject'
+            examples:
+              data:
+                value:
+                  email: 'foo@bar.com'
+                  password: 'password'
+                  channel_id: 1
+      responses:
+        '200':
+          description: Returns if the customer credentials provided are valid or not
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ValidateCustomerCredentialsResponseObject'
+              examples:
+                data:
+                  value:
+                    is_valid: true
+                    customer_id: 1
+        '422':
+          description: The *Customer* was not valid. This is the result of missing required fields or trying to edit a read only field. See the response for more details.
```

Comment:
Maybe just say `This is the result of missing required fields. See the response for more details.`